### ISSUE TITLE
Portable PDBs support

### ISSUE THREAD

**Issue by Fresa** [SymbolSource/SymbolSource#7]
I successfully uploaded a package but it cannot be found when trying to list it.

```
.\nuget.4.7.0.exe push C:\Users\Fres
a\Downloads\Topshelf.ApplicationHostBuilder.1.0.10.symbols.nupkg -source https://nuget.smbsrc.net/
Pushing Topshelf.ApplicationHostBuilder.1.0.10.symbols.nupkg to the symbol server (https://nuget.smbsrc.net/)...
  POST https://www.nuget.org/api/v2/package/create-verification-key/Topshelf.ApplicationHostBuilder/1.0.10
  OK https://www.nuget.org/api/v2/package/create-verification-key/Topshelf.ApplicationHostBuilder/1.0.10 1902ms
  PUT https://nuget.smbsrc.net/api/v2/package/
  OK https://nuget.smbsrc.net/api/v2/package/ 2401ms
Your package was pushed.
```

```
 .\nuget.4.7.0.exe list Topshelf.Appl
icationHostBuilder -PreRelease -AllVersions -source https://nuget.smbsrc.net/
No packages found.
```

I get it if it takes some time to index the package, but it has been hours now.

**Comment by JeremyTCD** [comment]
Seems like [Portable PDB](https://github.com/dotnet/core/blob/master/Documentation/diagnostics/portable_pdb.md)s aren't supported by SymbolSource.  Newtonsoft.Json had the same problem, their solution was to include the PDBs in the main package and to enable SourceLink: https://github.com/JamesNK/Newtonsoft.Json/issues/1666#issuecomment-399718552

**Comment by Fresa** [comment]
@JeremyTCD: Good find, thanks!

So the question should then be, when will SymbolSource support portable PDB's? 

Refering to the issue thread on Newtonsofts Github it seems they have used protable PDB's with SymbolSource for more than 3 years, and it is still not supported. Since portable PDB's is becoming the defacto standard with .NET, shouldn't supporting it be some what prioritized?

**Comment by JeremyTCD** [comment]
SymbolSource is kinda [dead](https://twitter.com/TripleEmcoder/status/960974350714359808) unfortunately. Nuget have their own [plans](https://github.com/NuGet/Home/wiki/NuGet-Package-Debugging-&-Symbols-Improvements#source-linking-with-nuget-packages-) for hosting symbols. SourceLink is probably the way to go right now, personally am just going to do what Newtonsoft is doing!

**Comment by TripleEmcoder** [comment]
You are correct - Portable PDBs are not supported at the moment. Any PRs moving us closer to such support are very welcome. If we don't get there with SymbolSource, nuget.org is going to fill the gap, as mentioned in the improvements spec.

The more pressing issue in my opinion is the lack of feedback from the submission process. Even with access to the server side of things it still wasn't straightforward to pinpoint the issue.
